docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "2"
+
+services:
+  runner:
+    image: lubien/project_euler
```

Comment:
Until @ryukinix creates an account we may use your image, is that ok?

Reply:
As estated in the PR, that's the plan.
I've already asked him to do it but there where some issues on the website.

Reply:
Yes, i'm in trouble to create an account on docker hub because some seems fucked on forms to sign up.